content/docs/rfcs/70/README.md

```diff
@@ -0,0 +1,236 @@
+
+$$ SPK = spk \cdot G $$
+
+SPK is a public key generated and stored at medium-term. 
+It is called a signed prekey because Bob also needs to store a public key certificate of SPK using IK. 
+Both signed prekey and the certificate must undergo periodic replacement, 
+a process that entails the generation of a fresh signed prekey. 
+After replacing the key, 
+Bob keeps the old private key of SPK for some interval, dependant on the implementation.
+This allows Bob to decrypt delayed messages. 
+It is important that Bob does not reuse SPKs. 
+This action is pivotal for ensuring forward secrecy, as these keys are integral for recalculating the shared secret employed in decrypting historical messages.
+
+It will be required to sign SPK for authentication. Following the specification of X3DH, one will use the digital signature scheme XEd448 and define:
+
+$$ SigSPK = XEd448(ik, Encode(SPK)) $$
+
+A final step requires the definition of a  _prekey bundle_  given by the tuple
+
+$$ prekey\_bundle = (IK, SPK, SigSPK, \{OPK_i\}_i) $$
+
+Where the different one-time keys OPK are points in X448 generated from a random integer $opk \in \mathbb{Z}_p$ and computed by performing
+
+$$ OPK = opk\cdot G $$
+
+Before sending an initial message to Bob, Alice will generate an AD vector as described in the documentation:
+
+$$ AD = Encode(IK_A)|| Encode(IK_B) $$
+
+Alice will also need to generate ephemeral key pairs (ek, EK) following the above mechanisms, that is: ek is a random integer modulo p, and EK is the associated public key obtained from the product
+
+$$ EK = ek \cdot G $$
+
+The function Encode() transforms an X448 public key into a byte sequence. 
+The recommended encoding consists of a single-byte constant to represent the type of curve, followed by little-endian encoding of the u-coordinate. 
+This is specified in the [RFC 7748](http://www.ietf.org/rfc/rfc7748.txt) on elliptic curves for security.
+
+## Using X3DH in Double Ratchet
+
+According to [Signal specifications](https://signal.org/docs/specifications/doubleratchet/) 
+this specification uses the double ratchet in combination with X3DH using the following data as initialization for the former:
+
+-   The SK output from X3DH becomes the SK input of the double ratchet. See section 3.3 of [Signal Specification](https://signal.org/docs/specifications/doubleratchet/) for a detailed description.
+-   The AD output from X3DH becomes the AD input of the double ratchet. See sections 3.4 and 3.5 of  [Signal Specification](https://signal.org/docs/specifications/doubleratchet/)  for a detailed description.
+-   Bob’s signed prekey SigSPKB from X3DH is used as Bob’s initial ratchet public key of the double ratchet.
+
+Once this initialization has been set, Alice and Bob can start exchanging messages with forward secrecy and authentication.
+
+## Specification as a Noise protocol
+
+X3DH has three phases:
+
+1.  Bob publishes his identity key and prekeys to a server, or dedicated smart contract.
```

Comment:
or network 


Maybe we can define the terminlogy "location" and specific that it may refer to a server, a smart contract or a decentralized system that enable storage and retrieval of data.